index.html

```diff
@@ -0,0 +1,110 @@
+                        <li class="github"><a href="" class="a"><i class="fa-brands fa-github"></i></a></li>
+                        <li class="facebook"><a href="" class="a"><i class="fa-brands fa-facebook"></i></a></li>
+                        <li class="anything"><a href="" class="a">Spanish</a></li>
+                    </ul>
+                </div>
+                <div class="mobile-logo">
+                    <button type="button" class="hamburger"><i class="fa-solid fa-bars"></i></button>
+                    <img src="./images/josue-logo.png" alt="josue-design" class="logo-img">
+                </div>
+                
+                <ul class="navigation">
+                    <li class="logo"><img src="./images/josue-logo.png" alt="josue-design" class="logo-img"></li>
+                    <li class="menu"><a href="#intro">Home</a></li>
+                    <li class="menu"><a href="#program">Program</a></li>
+                    <li class="menu"><a href="#speakers">Speakers</a></li>
+                    <li class="menu"><a href="#sponsor">Sponsors</a></li>
+                    <li class="about"><a href="./about.html">About</a></li>
+                </ul>
+            </div>
+        </header>
+        <div id="intro" class="intro">
+            <h2 class="hello">"Hello! Sharing world"</h2>
+            <h1 id="title" class="title">INTERNATIONAL BOOK FAIR IN JESUS MARIA 2022 </h1>
+            <p class="description">You are invited to the International Book Fair in Lima (IBF)
+                that will take place on Proceres park in Jesus Maria</p>
+            <h3 class="date">22 July, 2022</h3>
+            <p class="location">Lima, Perú</p>
+        </div>
+    </div>
+    <div id="program" class="main-program">
+        <h2 class="subtitle">Main Program</h2>
+        <div class="linea"></div>
+        <div class="div-list">
+            <div class="item-list">
+                <img src="./images/list1.png" alt="conference" class="images">
+                <h3 class="img-title">Opinion</h3>
+                <p class="img-des">Listen to the Speakers experts of the art of writing</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list2.png" alt="conversation" class="images">
+                <h3 class="img-title">Conversation</h3>
+                <p class="img-des">Share your ideas and opinions with other people in the place</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list3.png" alt="buy" class="images">
+                <h3 class="img-title">Store</h3>
+                <p class="img-des">The most variable selection of books</p>
+            </div>
+        </div>
+        <a href="#" class="square">See whole program</a>
+        <div class="space"></div>
+    </div>
+    <div id="speakers" class="featured-speakers">
+        <h2 class="speakers-title">Featured Speakers</h2>
+        <div class="linea"></div>
+        
+    </div>
```

Comment:
- [ ] The original design has 6 speakers. Kindly add more than 2 speakers.

- [ ] Please make sure you follow the design [layout](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=composition%20and%20space%20between%20elements.)
- [ ] Well done adding feature when click the showing more speakers. It is a little different from the original. It has to a red arrow and ıt is a button. Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/176969276-aa5c4328-09b8-448a-beec-a7e241450028.png)

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/176969312-e99bb36a-f00d-4a0f-a257-846f82bf0850.png)
____
### YOUR DESIGN MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969462-6c327db8-98da-423a-b045-3825fc45f034.png)
___
ORIGINAL MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969508-513aa76b-0a48-4f34-82a3-62af10e46dbc.png)